### PR TITLE
Replaced format statements with fstrings

### LINKED ISSUE #80: Clean up awkward PDB2PQR string formatting
Where appropriate, replace old printf-style formatting with more modern formatting methods.

### PR DESCRIPTION
**Pull request by intendo**
This closes #80

### CODE REVIEW

File: pdb2pqr/aa.py [hunk 1]
```diff
@@ -200,23 +200,16 @@ def rebuild_tetrahedral(self, atomname):
...
-                    ang = util.dihedral(
-                        cg1atom.coords,
-                        nextatom.coords,
-                        bondatom.coords,
-                        hcoords,
-                    )
                 else:
                     ang = util.dihedral(
-                        cbatom.coords,
+                        cg1atom.coords,
```
Comment (sobolevnrm) [inline pdb2pqr/aa.py:212]: Are you sure of this change?

File: pdb2pqr/debump.py [hunk 2]
```diff
@@ -126,7 +126,9 @@ def get_bump_score_atom(self, atom):
             )
             cutoff = atom_size + other_size
             if dist < cutoff:
-                bumpscore = bumpscore + 1000.0
+                bumpscore += 1000.0
+                # Also ignore if this is a donor/acceptor pair
+                pair_ignored = False
```
Comment (sobolevnrm) [inline pdb2pqr/debump.py:131]: Doesn't this mean the pair is always ignored?

**Review by sobolevnrm — CHANGES_REQUESTED** [review]

File: pdb2pqr/aa.py (hunk 1 above)
Comment (intendo) [reply pdb2pqr/aa.py:212]: Yes, it is a rewrite of the `if/elif/else` to just be an `if/else`. The first `else` is elevated to the `if` with the `or bondatom.name != "CG2"` clause. Then first `elif` becomes the second `else`.

File: pdb2pqr/debump.py (hunk 2 above)
Comment (intendo) [reply pdb2pqr/debump.py:131]: Sorry, I should have put a warning and a question for @sobolevnrm there. The `pair_ignored` is never initialized in `debump.py` so there has been a logic error in the `if pair_ignored` for some time. I don't know the correct logic to implement to determine if a pair gets ignored or not. The current implementation just always logs a debug message.

Comment (intendo) [reply pdb2pqr/debump.py:131]: Should we create an issue for this one?

**Comment by sobolevnrm** [comment]
I think @intendo rebutted most of my comments and resolved the conversations.  I can't keep up so I'm approving.

**Review by sobolevnrm — APPROVED** [review]

**Pull request merged**